crates/re_sdk/src/log_sink.rs

```diff
@@ -134,7 +134,16 @@ impl MemorySink {
     /// Create a new [`MemorySink`] with an associated [`RecordingStream`].
     #[inline]
     pub fn new(rec: RecordingStream) -> Self {
-        Self(MemorySinkStorage::new(rec))
+        Self(MemorySinkStorage::new(rec, None))
+    }
+
+    #[doc(hidden)]
+    #[inline]
+    pub fn new_with_flush_hook(
```

Comment:
I added a bunch of these `_with_flush_hook` methods which are `#[doc(hidden)]` because I don't know if we want this to be a part of the public API